### DISCUSSION TITLE
Dynamic import with SSR disabled render in server and throws error

### DISCUSSION THREAD

**Discussion by peterfritz** [vercel/next.js#23672] [Help] [answered]
I'm trying to completely redesign [my website](https://peterfritz.dev/?utm_source=github&utm_medium=github_nossr_question) and add i18n, hopefully making the site more useful, fun, and interactive.

I wrote a simple stars background in p5.js and tried using p5 inside next.js with [react-p5](https://github.com/Gherciu/react-p5). Unfortunately, even tho I used multiple dynamic imports disabling SSR for those components, it throws a server error `ReferenceError: window is not defined` that makes the build process fail. The weird thing is that in the development script, after a couple of seconds, the page reloads, the error disappears, and the code starts working perfectly.

To reproduce, you can use [this sandbox](https://codesandbox.io/s/nextjs-nossr-error-bsnny).

I'm not sure how to fix this and if this is a next.js issue or not, hoping you guys can help me with that.
Thanks in advance.

_Versions: next@10.1.3, react-p5@1.3.18_

**Comment by raffij** [comment, marked as answer]
Use `ssr: false` rather than `srr: false`

**Reply by peterfritz** [reply]
I'm ashamed, I can't believe I did that.
Thank you so much and I'm really sorry for wasting your time.
